tests/issues/issue-1953.php

```diff
@@ -11288,13 +11288,13 @@
     'issue-1953-JX' => [
         'ua' => 'Opera/9.80 (Android; Opera Mini/37.0.2254/113.82; U; en) Presto/2.12.423 Version/12.16',
```

Comment:
The tests that I modified in this file must've been missed when originally added. I've confirmed the changes made make sense based on the UA.